### PR TITLE
Rs 2022 07 19

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Ciel/Day 3/03_01_CIEL03_3_2B.txt [hunk 1]
```diff
@@ -1,52 +1,59 @@
...
 -- ……死んでしまえるような、気がする。
- ...The feeling I'll... die.
+The feeling I'll... die.
 }
 [sha:2dfe5b44fe9aa69639eb250cae5ad8a7dfa993c1]{
 -- Page 130, Offset 5173.
 -- ザザ鳴りの神経、ザラついた体温。
-The nerves picking up %{i}that sound%{/i}, my coarse body temperature.
+// Reading ザザ as a pins and needles/numbness deal
+// A _gritty_ body temperature?? Goosebumps? Or cooled to the point of 'coagulating',
```
Comment (Avuxo) [inline script/Ciel/Day 3/03_01_CIEL03_3_2B.txt:53]: I mean... that's what it says. double check with petrikow, but this is _very_ liberally tl'd

File: script/Ciel/Day 3/03_01_CIEL03_3_2B.txt [hunk 2]
```diff
@@ -212,32 +223,33 @@ I hear someone call out my name.
...
 -- Page 138, Offset 5206.
 -- ……顔をあげる。
-......I look up
+...I look up.
 }
 [sha:27e37a8c85b1206f00e65a7bfd8b7ce33c39a21f]{
 -- Page 138, Offset 5207.
 -- そこには、もう何十年かぶりに思える、先輩の姿があった。
- There, for what feels like the first time in decades, I see Senpai.
+Standing there in front of me is Senpai.
```
Comment (Avuxo) [inline script/Ciel/Day 3/03_01_CIEL03_3_2B.txt:231]: revert

**Review by Avuxo — APPROVED** [review]

File: script/Ciel/Day 3/03_01_CIEL03_3_2B.txt [hunk 3]
```diff
@@ -272,48 +285,49 @@ It's a voice I heard only a few hours ago, but now it sounds so far away.
...
 -- Page 147, Offset 5225.
 -- 言うが早いか、先輩は俺の腕を掴んで、強引にベンチから立ち上がらせた。
-Senpai grabs my arm and forces me up from the bench.
+Before she's even finished speaking, Senpai has grabbed me by the arm and hoisted me off the bench.
 }
 [sha:7af64fa067a0dd2190c5ca3c5a763dfc376dc30c]{
 -- Page 148, Offset 5226.
 -- 「わたしの傘を貸しますから、すぐに家に帰って体を温めてください。本当に命にかかわります」
-"I'll lend you my umbrella, but you must go home and warm yourself at once. This is seriously a matter of life and death."
+"I'll lend you my umbrella, but you need go home and warm yourself up immediately.
```
Comment (yk1512) [inline script/Ciel/Day 3/03_01_CIEL03_3_2B.txt:329]: need to

File: script/Ciel/Day 3/03_01_CIEL03_3_2B.txt (hunk 2 above)
Comment (yk1512) [reply script/Ciel/Day 3/03_01_CIEL03_3_2B.txt:231]: I like ross' version better, sounds more natural to me.

**Review by yk1512 — APPROVED** [review]

File: script/Ciel/Day 3/03_01_CIEL03_3_2B.txt (hunk 2 above)
Comment (Avuxo) [reply script/Ciel/Day 3/03_01_CIEL03_3_2B.txt:231]: idk, old version reads more naturally to me and doesn't have the pause.

**Pull request merged**